### DISCUSSION TITLE
How to Change Progress Bar Color

### DISCUSSION THREAD

**Discussion by macellinoss275** [livewire/livewire#6619] [2. Show and tell]
I saw a question on Laracasts and I feel like I should share my answer here because people are probably gonna need it. In Livewire 2, we need to manually add @livewireStyles and @livewireScripts. In Livewire 3 however, those assets are injected by default by Livewire.

Now go to the console in your browser, and expand the head tag, you'll then see 2 Livewire styles tags like the image below:
![image](https://github.com/livewire/livewire/assets/138418079/b6f6f2af-eae4-482a-80f2-35d9e78befae)

If you want to change the color of the progress bar, expand the second style tag. You'll then see a bunch of codes like the image below:
![image](https://github.com/livewire/livewire/assets/138418079/5fb84369-fbbf-4569-a0fa-04fb28fed560)

So yeah all you need to do now is basically to create your own CSS that targets the id and class that Livewire NProgress style uses, something like this:
```
        <style>
            #nprogress .bar {
                background: red;
            }
        </style>
```

**Comment by sajjadhossainshohag** [comment]
Trying this way. Use important.

`<style>
           #nprogress .bar {
                background: red !important;
           }
</style>`

**Comment by ezehkingsleyuchenna** [comment]
Add this to your style
```
:root {
      --livewire-progress-bar-color: red !important;
  }

**Comment by cjmellor** [comment]
You can do it in the Livewire config file

```
    /*
    |---------------------------------------------------------------------------
    | Navigate (SPA mode)
    |---------------------------------------------------------------------------
    |
    | By adding `wire:navigate` to links in your Livewire application, Livewire
    | will prevent the default link handling and instead request those pages
    | via AJAX, creating an SPA-like effect. Configure this behavior here.
    |
    */

    'navigate' => [
        'show_progress_bar' => true,
        'progress_bar_color' => '#2299dd', // <---
    ],
```
